docs/content/modeling/roles-and-permissions.mdx

```diff
@@ -94,37 +93,37 @@ In addition, you will need to know the following:
 
 ### Direct Access
 
```

Comment:
Review the capitalization section of our style guide & apply to all headings and sub-headings below:

https://oktawiki.atlassian.net/wiki/spaces/DOCS/pages/2544472397/Document+Structure#Capitalization